python/ray/train/v2/_internal/execution/context.py

```diff
@@ -287,3 +385,7 @@ def set_train_context(context) -> None:
     global _train_context
     with _context_lock:
         _train_context = context
+
+
+def is_running_local_testing() -> bool:
+    return isinstance(get_train_context(), LocalTestingTrainContext)
```

Comment:
I'm thinking it might be cleaner to just add an `is_local()` method to the `TrainContext` so that it is more explicit to the user what the "local" refers to.

The user code would look like:

```python
def train_func():
    if ray.train.get_context().is_local():
        ...
```